operations/mimir-mixin/dashboards/writes.libsonnet

```diff
@@ -156,33 +156,15 @@ local filename = 'mimir-writes.json';
         $.timeseriesPanel('Per %s p99 latency' % $._config.per_instance_label) +
         $.perInstanceLatencyPanelNativeHistogram('0.99', $.queries.distributor.requestsPerSecondMetric, $.jobSelector($._config.job_names.distributor) + [utils.selector.re('route', '/distributor.Distributor/Push|/httpgrpc.*|%s' % $.queries.write_http_routes_regex)])
       )
-      .addPanelIf(
```

Comment:
Note to reviewers: this panel was the same as the one in the ruler, so I moved it to a shared function.